source/time_tables/previous_passages.cpp

```diff
@@ -59,25 +59,28 @@ previous_passages(const std::string &request,
     std::remove_if(handler.journey_pattern_points.begin(),
                    handler.journey_pattern_points.end(), vis.predicate);
 
-    auto passages_dt_st = get_stop_times(handler.journey_pattern_points,
-                            handler.date_time, handler.max_datetime,
-                            nb_stoptimes, data, disruption_active, accessibilite_params,
-                            false);
+    const bool is_on_departures(vis.api_pb == pbnavitia::PREVIOUS_DEPARTURES);
```

Comment:
why not `const bool is_on_departures = vis.api_pb == pbnavitia::PREVIOUS_DEPARTURES; `